### ISSUE TITLE
Load DataSet Slow GTX1050Ti

### ISSUE THREAD

**Issue by Suaig** [fastai/fastai#2493]
**Describe the bug**

BUG is about train when program load data.
load data is very slow.I say that training is divided into three parts.
part1:LoadData is use 50 sec
part2:GPU train model,use about 55 sec
part3:vaild (Load,and predic) LoadData slow too

when i use 'paperspace'(a ML service https://www.paperspace.com/) .it work well.
Keras is work too(USE GPU)

question:why loaded dataset cost too many time in my PC.

thinks FastAI team!

**Provide your installation details**

Info:
```text
=== Software === 
python        : 3.6.7
fastai        : 1.0.60
fastprogress  : 0.2.2
torch         : 1.4.0
nvidia driver : 442.19
torch cuda    : 10.1 / is available
torch cudnn   : 7501 / is enabled

=== Hardware === 
nvidia gpus   : 1
torch devices : 1
  - gpu0      : 4096MB | GeForce GTX 1050 Ti

=== Environment === 
platform      : Windows-10-10.0.18362-SP0
conda env     : pytroch_py36_env
python        : C:\ProgramData\Anaconda3\envs\pytroch_py36_env\python.exe
sys.path      : C:\ProgramData\Anaconda3\envs\pytroch_py36_env\python36.zip
C:\ProgramData\Anaconda3\envs\pytroch_py36_env\DLLs
C:\ProgramData\Anaconda3\envs\pytroch_py36_env\lib
C:\ProgramData\Anaconda3\envs\pytroch_py36_env

C:\ProgramData\Anaconda3\envs\pytroch_py36_env\lib\site-packages
C:\ProgramData\Anaconda3\envs\pytroch_py36_env\lib\site-packages\win32
C:\ProgramData\Anaconda3\envs\pytroch_py36_env\lib\site-packages\win32\lib
C:\ProgramData\Anaconda3\envs\pytroch_py36_env\lib\site-packages\Pythonwin
C:\ProgramData\Anaconda3\envs\pytroch_py36_env\lib\site-packages\IPython\extensions
C:\Users\MY\.ipython
```

Please make sure to include opening/closing ``` when you paste into forums/github to make the reports appear formatted as code sections.

**To Reproduce**

USE GTX1050TI when Train.
Win10 1909 workstation
Python 3.6.5 CUDATOOL 10.2 

**Expected behavior**

**Screenshots**

![pg1](https://user-images.githubusercontent.com/24272990/74600278-f65a8f00-50c9-11ea-96fa-8ff3af6990da.jpg)
![pg2](https://user-images.githubusercontent.com/24272990/74600280-f9ee1600-50c9-11ea-91d9-bb9d2417e24e.jpg)
It use 50S to loaddata Then train
![p3](https://user-images.githubusercontent.com/24272990/74600289-433e6580-50ca-11ea-8cf7-8a70b073f39a.jpg)
**Additional context**

**Comment by Suaig** [comment]
code
'''
from fastai.vision import models, URLs, ImageDataBunch, cnn_learner, untar_data, accuracy
path = untar_data(URLs.CIFAR,dest='./')  # 下载数据集，这里只是MNIST的子集，只包含3和7的图像,会下载并解压（untar的命名原因）到/root/.fastai/data/mnist_sample（如果你是root用户）下，包含训练数据，测试数据，包含label的csv文件
data = ImageDataBunch.from_folder(path)  # 利用ImageDataBunch读取文件夹，返回一个ImageDataBunch对象
learn = cnn_learner(data, models.vgg16_bn, metrics=accuracy)  # 构建cnn模型，使用resnet18预训练模型
learn.fit_one_cycle(4)
'''

**Comment by sgugger** [comment]
This is linked to multiprocessing on Windows with PyTorch not really working. You should bring the issue to them (I think several issues are opened for that problem). As far as I know, there is nothing much we can do on our side...
